identity-service/src/notifications/sendNotifications/index.js

```diff
@@ -57,24 +57,63 @@ function getUserIdsToNotify(notifications) {
 const getUserNotificationSettings = async (userIdsToNotify, tx) => {
   const userNotificationSettings = {}
 
+  // fetch user's with registered browser devices
+  const userNotifSettingsMobile = await models.sequelize.query(
+    `
+      SELECT * 
+      FROM "UserNotificationMobileSettings" settings
+      WHERE 
+        settings."userId" in (:userIds) AND 
+        settings."userId" in (
+          SELECT "userId"
+          FROM "NotificationDeviceTokens" device
+          WHERE
+            device.enabled AND
+            device."deviceType" in ('ios', 'android')
+          )
+    `,
+    {
+      replacements: {
+        userIds: userIdsToNotify
+      },
+      model: models.UserNotificationMobileSettings,
+      mapToModel: true
+    }
+  )
+
   // Batch fetch mobile push notifications for userIds
-  const mobileQuery = {
-    where: { userId: { [models.Sequelize.Op.in]: userIdsToNotify } },
-    transaction: tx
-  }
-  const userNotifSettingsMobile =
-    await models.UserNotificationMobileSettings.findAll(mobileQuery)
   userNotifSettingsMobile.forEach((settings) => {
     userNotificationSettings[settings.userId] = { mobile: settings }
   })
 
   // Batch fetch browser push notifications for userIds
-  const browserPushQuery = {
-    where: { userId: { [models.Sequelize.Op.in]: userIdsToNotify } },
-    transaction: tx
```

Comment:
new code omits `transaction`... which is probably okay... but maybe we want to keep that context?

Reply:
I don't think we need bc this is a read, but good point, can include